modules/ossm-migrating-a-multitenant-deployment.adoc

```diff
@@ -0,0 +1,179 @@
+====
+
+. Restart the workloads:
++
+.. You can restart all the workloads at once so that the new pods will be injected with the {SMProduct} 3.0 proxy. For this example with bookinfo, run the following command:
++
+[source,terminal]
+----
+$ oc rollout restart deployments -n bookinfo
+----
+
+.. Alternatively, you can restart each workload individually by running the following command for each workload:
++
+[source,terminal]
+----
+$ oc rollout restart deployments productpage-v1 -n bookinfo
+----
+
+. Wait for the `productpage` application to restart by running the following command:
++
+[source,terminal]
+----
+$ oc rollout status deployment productpage-v1 -n bookinfo
+----
+
+=== Validate Workload Migration
+
+. Check that your workload is connected to the new controlplane.
+
+.. See which proxies are still connected to the 2.6 controlplane with `istioctl` by running the following command:
++
+[source,terminal]
+----
+$ istioctl ps --istioNamespace istio-system-tenant-a --revision basic
+----
++
+Here `basic` should be the name of your `ServiceMeshControlPlane`:
++
+.Example output
+
+[source,terminal]
+----
+   NAME                                              CLUSTER        CDS        LDS        EDS        RDS          ECDS         ISTIOD                            VERSION
+   details-v1-7b49464bc-zr7nr.bookinfo               Kubernetes     SYNCED     SYNCED     SYNCED     SYNCED       NOT SENT     istiod-basic-6c9f8d9894-sh6lx     1.20.8
+   ratings-v1-d6f449f59-9rds2.bookinfo               Kubernetes     SYNCED     SYNCED     SYNCED     SYNCED       NOT SENT     istiod-basic-6c9f8d9894-sh6lx     1.20.8
+   reviews-v1-686cd989df-9x59z.bookinfo              Kubernetes     SYNCED     SYNCED     SYNCED     SYNCED       NOT SENT     istiod-basic-6c9f8d9894-sh6lx     1.20.8
+   reviews-v2-785b8b48fc-l7xkj.bookinfo              Kubernetes     SYNCED     SYNCED     SYNCED     SYNCED       NOT SENT     istiod-basic-6c9f8d9894-sh6lx     1.20.8
+   reviews-v3-67889ffd49-7bhxn.bookinfo              Kubernetes     SYNCED     SYNCED     SYNCED     SYNCED       NOT SENT     istiod-basic-6c9f8d9894-sh6lx     1.20.8
+----
+
+.. Check which proxies have been migrated to the new {SMProduct} 3.0 controlplane by running the following command:
++
+[source,terminal]
+----
+istioctl ps --istioNamespace istio-system-tenant-a --revision istio-tenant-a
+----
++
+.Example output
++
+[source, terminal]
+----
+NAME                                         CLUSTER        CDS        LDS        EDS        RDS        ECDS     ISTIOD                      VERSION
+productpage-v1-7745c5cc94-wpvth.bookinfo     Kubernetes     SYNCED     SYNCED     SYNCED     SYNCED              istiod-5bbf98dccf-n8566     1.24.3
+----
+
+. Verify the `bookinfo` application is still working correctly by running the following command:
```

Comment:
Trying to make it more obvious to readers that they will need to verify their own applications are working correctly and the exact commands they will run to do that will be different. The following command only applies to the bookinfo example.
```suggestion
. Verify your application is still working correctly. For the `bookinfo` application, run the following command:
```